src/clj/chronograph/db/core.clj

```diff
@@ -0,0 +1,11 @@
+(ns chronograph.db.core
+  (:require [mount.core :refer [defstate]]
+            [next.jdbc :as jdbc]
+            [next.jdbc.date-time :as jdbc-date-time]
+            [chronograph.config :as config]))
+
+(jdbc-date-time/read-as-instant)
```

Comment:
Can we put this in the defstate start fn?

Reply:
This just extends a protocol, it's fine to have it at the top level.